src/saving/SaveList.cs

```diff
@@ -38,12 +39,25 @@ public class SaveList : ScrollContainer
     [Export]
     public NodePath LoadInvalidSaveDialogPath;
 
+    [Export]
+    public NodePath LoadInvalidSaveOkPath;
+
+    [Export]
+    public NodePath LoadInvalidSaveCancelPath;
+
+    [Export]
+    public NodePath LoadInvalidSaveOpenLogPath;
```

Comment:
And if the stuff is logged in the main log file, there is no need for this extra button.

Reply:
In fact, the button doesn't even work with log.txt, because it is used by the thrive process.
Should I remove the custom dialog and revert it to a confirmationdialog?

Reply:
yes. I don't think the custom button is needed so that is fine.